README.md

```diff
@@ -33,9 +33,9 @@ To develop ontop of `0x-api`, follow the following instructions:
 
 4. Open the `.env` file and edit the following fields. Defaults are defined in `config.ts`/`config.js`. The bash environment takes precedence over the `.env` file. If you run `source .env`, changes to the `.env` file will have no effect until you unset the colliding variables.
 
-    - `NETWORK_ID` -- the network you'd like your relayer to run on (e.g: `1` -> mainnet, `42` -> Kovan, 3 -> Ropsten, etc...). Defaults to `42`
+    - `NETWORK_ID` -- the network you'd like your API to run on (e.g: `1` -> mainnet, `42` -> Kovan, 3 -> Ropsten, etc...). Defaults to `42`
```

Comment:
If we are removing references to the word "relayer" here should we should also change a couple references in `http_service.ts` and `errors.ts`